tools/utils/eachline.ts

```diff
@@ -1,19 +1,22 @@
-import split from "split2";
-import pipe from "multipipe";
-import { Transform } from "stream";
+const split = require("split2");
```

Comment:
can't you keep using import by doing `import * as split from "split2"` for example?

Reply:
Hi, thanks for commenting!

If I do that, I get a TS error like this:
```
Could not find a declaration file for module 'split2'. '/Users/newman/Code/meteor/tools/node_modules/split2/index.js' implicitly has an 'any' type.
  Try `npm install @types/split2` if it exists or add a new declaration (.d.ts) file containing `declare module 'split2';`
```

This would be easy to resolve if I had any idea of how to install `@types/split2` in this particular repo. But, `meteor/meteor` is unique and unfamiliar to me with its own special packaging, so I'm not sure the "proper" way to add a new NPM package here (where is my `package.json`???) -- hence my question in the PR body.

Reply:
Oops, sorry I didn't read the original PR body, just was perusing the code changes! Great contribution!

I'm not an expert on Meteor core development but I am endless curious so here's some of my findings...

Normally you should be able to add an npm dependency for any Meteor  package by adding it to the `package.js` and using `Npm.depends` to specify the specific version.

But using good ole `grep -r` I noticed that `split2` is defined in a couple places where it seems like Meteor generates its top-level `package.json`.

```
scripts/dev-bundle-tool-package.js
53:    split2: "2.2.0",

scripts/dev-bundle-server-package.js
24:    split2: "2.1.1",

tools/utils/eachline.js
1:import split from "split2";
```

Presumably you could add that there and regenerate the dev bundle?

Reply:
Thanks for the avenue of investigation! I was kind of hoping that someone who is an expert at Meteor core development would swoop in with a guideline on installing `@types/` NPM packages for this TypeScript conversion effort, haha, since I think it will be helpful for many of the files under `meteor/tools`.

(Related: #10613)